### PR TITLE
Fixed 404 URL issue for the 'User guides' link

### PR DESCRIPTION
**Pull request by NamrataChangani**
## This PR is a:
- [ ] Content fix or rewrite
 

## Summary
 Fixed 404 URL issue for the 'User guides' link
 

## Additional information
N/A

### CODE REVIEW

File: guides/v2.0/cloud/bk-cloud.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ To better understand {{site.data.var.ee}}, your plan, architecture, and workflow
 ## Learn more about Magento 2 {#magento2}
 If you would like to learn more about Magento 2, see the following resources:
 
-*	[All documentation home page](https://magento.com/help/documentation){:target="_blank"}
```
Comment (mhauri) [inline guides/v2.0/cloud/bk-cloud.md]: This commit is a already in PR #2036

**Comment by shrielenee** [comment]
Hey @NamrataChangani - You have submitted a revision for v2.0 docs, which has reached EOL (end of life). We are no longer updating that doc set. The change you're requesting, however, applies to v2.1+. 

Can you revise your PR to include the change (`guides/v2.0/cloud/bk-cloud.md`) for v2.1 (the rest of the versions are symlinked) and revert your change for the `guides/m1x/index.html` file, as it's already been address in PR #2036? After that, please push back up your revisions here. Thanks!

File: guides/m1x/index.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
 ---
 
 <p>Welcome to the home page for Magento 1.x documentation for installation, configuration, developers, patches, and more. Here you'll find articles that were formerly located on <code>http://magentocommerce.com/knowledge-base</code>.</p>
-<p>For information you don't find here&mdash;including Release Notes&mdash;see <a href="http://magento.com/help/documentation" target="_blank">the User Guide page</a>.</p>
+<p>For information you don't find here&mdash;including Release Notes&mdash;see <a href="https://magento.com/resources/technical" target="_blank">the User Guide page</a>.</p>
```
Comment (shrielenee) [inline guides/m1x/index.html]: Please revert this change - already addressed in another merged PR.

**Comment by shrielenee** [comment]
Hey @NamrataChangani  - I'm going to go ahead and close this PR since it addresses changes that have been addressed in another PR. See comments above for info about this change for v2.1, in case you'd like to make changes to that version.
